src/platforms/mod.rs

```diff
@@ -39,6 +39,40 @@ pub enum TargetEnv {
     MUSL,
 }
 
+///
+/// The list of target triples used as constants.
+///
+/// It must be in the lowercase.
+///
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+pub enum TargetTriple {
```

Comment:
Please move this entity to another file.

Reply:
And `TargetEnv` as well.
We shouldn't end up with files of 1000 lines with dozens of declarations.

Reply:
It should be completed. Please, take a look when you have a sec.